man/man1/zhack.1

```diff
@@ -71,6 +71,44 @@ feature in \fIpool\fR.
 .IP
 The \fB\-m\fR switch indicates that the \fIguid\fR feature is now
 required to read the pool MOS.
+.LP
+.BI "scrub " "pool"
+.IP
+Import and scrub a pool, entirely from userland.  This is mostly useful for
+developers and is substantially less friendly than scrubbing an imported
+pool.  In particular, there is no status reporting mechanism, and so
+any discovered errors are not easily identified.  For a very, very niche
+audience, this tool may have another purpose: the in-order scan code that
+is not yet in any stable ZFS-on-Linux release may be able to resilver pools
+much faster than the in-kernel ZFS implementation.
+.IP
+The \fB\-D class\fR option specifies the DDT class(es) to scan at most once.
+Setting class to 0 will limit special handling to dedup-ditto blocks.  The
+value 1 will ensure that the scan will visit all deduplicated blocks with
+refcounts greater than one at most once, but will require that the DDT be
+frequently accessed during scan.  The value 2 will cause the scan to walk all
+DDT entries, including those with refcount one, and will then not need to
+probe the DDT during the residual scan of non-deduplicated blocks.  The
+default behavior, at the time of this writing, is as if the value 1 had been
+specified.  Pass \fB\-E\fR to attempt to change the DDT class on an existing
+scrub; the tool will refuse to execute any unsafe change (i.e., downwards).
+.IP
+The \fB\-n\fR option will prevent the tool from launching a new scrub or
+resilver if one was not in progress at the time of export.  Contrawise,
+\fB\-R\fR will restart any existing scrub, and \fB\-r\fR will start a resilver
+rather than a scrub.
```

Comment:
Would this be a little more concise summary of the intended behavior of these options.

**-n** - Only resume a scrub if one was in progress, never start a new one.
**-R** - Cancel any running scrub/resilver and start a new scrub.
**-r** - Cancel any running scrub/resilver and start a new resilver.

Reply:
Not quite; took another stab.